fast_and_reliable.py

```diff
@@ -0,0 +1,37 @@
+import math
+import time
+
+
+def average_runtime(words):
```

Comment:
Document all functions, add type hints and return types to all of them -
![Capture](https://user-images.githubusercontent.com/13467749/159866247-6874e703-fbec-46da-a4b8-cb03331b83e5.JPG)

Reply:
read about return types here https://docs.python.org/3/library/typing.html